client/packages/common/src/ui/forms/JsonForms/components/Text.tsx

```diff
@@ -9,12 +9,14 @@ import {
 export const stringTester = rankWith(3, schemaTypeIs('string'));
 
 const UIComponent = (props: ControlProps) => {
-  const { data, handleChange, label, path } = props;
+  const { data, handleChange, label, path, description, errors } = props;
```

Comment:
offroad: I thought to show the error, if there is one, and have also added the description as placeholder text.
This allows for

<img width="414" alt="Screen Shot 2022-07-28 at 5 21 29 PM" src="https://user-images.githubusercontent.com/9192912/181426815-4b3c8703-b09d-4fa4-93f4-18e461c2588e.png">


and 

<img width="432" alt="image" src="https://user-images.githubusercontent.com/9192912/181426777-888486ab-9aa7-4497-8cc8-ac3504e76471.png">